index.html

```diff
@@ -0,0 +1,753 @@
+                            <h5>VENTURE CAPITAL</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Information Technology, +2</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>
+                                    Development Bank of Wales
+                                </span></i></a>
+                            <div class="border">
+                                <p>Agxio secures £750k investment funding <a href="#" class="faded-out">UKTN</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="sponsor">
+                    <div class="sponsor-image">
+                        <img src="assets/images/sponsor2.png" alt="sponsor image">
+                    </div>
+                    <div class="sponsor-details">
+                        <h3>Ahana</h3>
+                        <div class="sponsor-assets">
+                            <h4>$2.5M</h4>
+                            <h5>Seed Fund</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Information Technology</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>GV</span></i></a>
+                            <div class="border">
+                                <p>Ahana gets seed funding <a href="#" class="faded-out">ZDNet</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="sponsor">
+                    <div class="sponsor-image">
+                        <img src="assets/images/sponsor3.jpeg" alt="sponsor image">
+                    </div>
+                    <div class="sponsor-details">
+                        <h3>Zeleros</h3>
+                        <div class="sponsor-assets">
+                            <h4>$7M</h4>
+                            <h5>Series A</h5>
+                        </div>
+                        <div class="sponsor-cards">
+                            <a href="#" class="sponsor-tag"><i class="fas fa-tag"><span> Design, +3</span></i></a>
+                            <a href="#" class="sponsor-tag"><i class="fas fa-briefcase">
+                                <span>Altran, +6</span></i></a>
+                            <div class="border">
+                                <p>Zeleros raises €7m <a href="#" class="faded-out">UKTN</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
```

Comment:
There are still responsiveness issues in this section. Try and make the medium view (tablet view ) to look like the [original](https://thenextweb.com/). Your `.sponser` class should span the whole width at medium view check the [original](https://thenextweb.com/).